spec/factories/ext_management_system.rb

```diff
@@ -391,7 +391,10 @@
           :aliases => ["manageiq/providers/embedded_ansible/automation_manager"],
           :class   => "ManageIQ::Providers::EmbeddedAnsible::AutomationManager",
           :parent  => :automation_manager do
-    provider :factory => :provider_embedded_ansible
+    provider {
+      raise "Deprecated:  This Factory is deprecated!  Use :provider_embedded_ansible" \
```

Comment:
Minor but "deprecated" implies it will continue to work temporarily but will be removed in the future.  In this case, you want it to straight-up fail now, in which case I would drop that phrasing.

Reply:
Fair enough.  I was curious when we were going to get rid of it as well, but probably just raising the error sans-deprecation warning is probably good enough.

Reply:
Better?

Reply:
Yup looks good.